theme/styles/snippets/product-grid-item.scss

```diff
@@ -0,0 +1,43 @@
+.ProductGridItem {
+  display: grid;
+
+  //Aspect ratio 
+  &::before {
+    content: "";
+    display: block;
+    height: 0;
+    width: 0;
+    padding-bottom: calc(480/499 * 100%);
+    grid-area: 1 / 1 / 2 / 2;
+  }
+
+  &__inner-container {
+    grid-area: 1 / 1 / 2 / 2;
+    width: 100%
+  }
+
+  &__image {
+    width: 60%;
+    max-height: 13rem;
+
+    @include media('xs-up') {
+      max-height: 22rem;
+      padding: 1rem 0;
+    }
+
+    @include media('md-up') {
+      max-height: 11rem;
+      padding: 1rem 0;
+    }
```

Comment:
Any reason we can't apply the padding here at just the smaller breakpoint? Seem like a lot of duplication adding it at every BP